openslides/core/static/templates/index.html

```diff
@@ -9,6 +9,7 @@
 <link rel="stylesheet" href="static/css/openslides-libs.css">
 <link rel="stylesheet" href="static/css/app.css">
 <script src="static/js/openslides-libs.js"></script>
+<script src="static/ckeditor/ckeditor.js"></script>
```

Comment:
It works if we:

1. Add this  code before openslides-lib.js is loaded:
```<script>
var CKEDITOR_BASEPATH= "/static/ckeditor/";
</script>
```
2. Add a lot of files from the ckeditor folder to our static folder.

Im not sure what to do.